reducer-todo/src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import TodoList from './components/TodoList';
+import './App.css';
+
+// Import TodoList and return it (display it) in App
+function App() {
+  return (
+    <div className="App">
+      <TodoList />
+    </div>
```

Comment:
I know the instructions say to not do everything inside App.js, I think what they mean is to have multiple components and limited amounts of logic written on the App.js itself.  I think if you put all the logic you had inside TodoList inside App.js it would be a very good amount of code inside app.js